charts/ExploreView.tsx

```diff
@@ -1,9 +1,21 @@
 import * as React from "react"
 import * as ReactDOM from "react-dom"
+import { observable, computed, autorun, IReactionDisposer } from "mobx"
+import { observer } from "mobx-react"
+import { extend } from "lodash"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core"
+import {
+    faChartLine,
+    faChartBar,
+    faChartArea,
+    faMap
+} from "@fortawesome/free-solid-svg-icons"
```

Comment:
This is the case where webpack doesn't do a good job treeshaking – you can see that the build has now grown from 1.8MB to 2.4MB. 

You can import the icons like this to avoid the problem:
```js
import { faChartLine } from "@fortawesome/free-solid-svg-icons/faChartLine"
import { faChartBar } from "@fortawesome/free-solid-svg-icons/faChartBar"
import { faChartArea } from "@fortawesome/free-solid-svg-icons/faChartArea"
import { faMap } from "@fortawesome/free-solid-svg-icons/faMap"
```

Reply:
Fixed this! (Hope you don't mind me commiting, I only wanted to merge this into my branch quicker so I can build on it without conflicts)

Reply:
I see. Cool, thanks for the tip